internal/project/project.go

```diff
@@ -46,7 +46,7 @@ type snapshot struct {
 func (s *snapshot) GetLineMap(fileName string) *ls.LineMap {
 	file := s.program.GetSourceFile(fileName)
 	scriptInfo := s.project.host.GetScriptInfoByPath(file.Path())
-	if file.Version == scriptInfo.Version() {
+	if s.project.getFileVersion(file, s.program.Options()) == scriptInfo.Version() {
```

Comment:
Doesn't the left side of this just return the script info too?

Reply:
It returns the version in the document registry's entry for that file. Do you mean that somehow that version is gonna end up always being the script info version? If so, do you have a scenario in mind? I can't really see it.

Reply:
I think I'm just wrong, the doc registry maps paths/options to source files, not scriptinfos.